firebase.json

```diff
@@ -11,6 +11,19 @@
         "source": "**",
         "destination": "/index.html"
       }
-    ]
+    ],
+    "headers": {
```

Comment:
Accroding to the doc, headers field should be array.
https://firebase.google.com/docs/hosting/full-config#headers
```
"hosting": {
  // ...

  // Applies a CORS header for all font files
  "headers": [ {
    "source": "**/*.@(eot|otf|ttf|ttc|woff|font.css)",
    "headers": [ {
      "key": "Access-Control-Allow-Origin",
      "value": "*"
    } ]
  } ]
}
```